@types/modules/pm/prize.d.ts

```diff
@@ -0,0 +1,26 @@
+declare interface PMPrizeUpdateRequest {
+    prizeDisplayName?: string;
+    prizePrice?: string;
+    prizeListingName?: string;
+    prizeUrl?: string;
+    prizeasin?: string;
+  }
+  
+  declare interface PMPrizeCreateRequest extends PMPrizeUpdateRequest {
+    prizeId: string;
+    prizeDisplayName: string;
+    prizePrice: string;
+    prizeListingName: string;
+    prizeUrl: string;
+    prizeasin: string;
+
+  }
+  
+  declare interface PMPrize extends PMPrizeCreateRequest {
+    
+  }
+  
+  declare interface PMPrizeList {
+    Prize: PMPrize[];
```

Comment:
This should be lower case and plural.
```suggestion
    prizes: PMPrize[];
```